tests/e2e/blocks/related-articles.spec.js

```diff
@@ -32,10 +32,10 @@ test('Test Related Articles block', async ({page, admin, editor}) => {
   await page.waitForTimeout(1000); // letting metabox post query finish
 
   await page.goto(postUrl);
-  const relatedSection = await page.locator('[data-render="planet4-blocks/articles"]');
+  const relatedSection = await page.locator('.p4-query-loop');
```

Comment:
Technically we could have another Query Loop block in the post, no? I'm wondering if we need to refine this test to make sure that we are targeting the one for the Related Articles section 🤔